### PR TITLE
All details pages modified to not allow select options and highlight(BOS-46)

### PR DESCRIPTION
**Pull request by pbattu123**
BOS-46 issue fix

### CODE REVIEW

**Comment by mneynens** [comment]
Looks goodl

File: bos_mint/views.py [hunk 1]
```diff
@@ -947,11 +947,13 @@ def genericUpdate(formClass, selectId, removeSubmits=False):
     parentId = None
     if selectId:
         parentId = utils.getParentTypeGetter(typeName)(selectId)
-
+    print("**** Views.py **** selectId, parentId, typeName",\
```
Comment (sschiessl-bcp) [inline bos_mint/views.py]: Please use logger, don't print directly to console.

File: bos_mint/forms.py [hunk 2]
```diff
@@ -248,6 +295,26 @@ def update(self, selectedId):
             selectedId,
             InternationalizedString.parseToList(self.name), self.sport.data)
 
+class EventGroupFormDetails(FlaskForm):
+    sport = TextField("Sport", render_kw={'readonly': True}, validators=[Optional()])
+    name = FormField(TranslatedFieldFormDetails, render_kw={'readonly': True})
+    submit = SubmitField("Submit")
+
+    print(" ** forms.py 256 ** " )
```
Comment (sschiessl-bcp) [inline bos_mint/forms.py]: Remove debug prints for release

File: bos_mint/forms.py [hunk 3]
```diff
@@ -248,6 +295,26 @@ def update(self, selectedId):
             selectedId,
             InternationalizedString.parseToList(self.name), self.sport.data)
 
+class EventGroupFormDetails(FlaskForm):
```
Comment (sschiessl-bcp) [inline bos_mint/forms.py]: In general it is not a good idea to introduce code duplication to extend logic. 

Refine existing class or introduce a derivation

**Comment by sschiessl-bcp** [comment]
@pbattu123

File: bos_mint/views.py (hunk 1 above)
Comment (pbattu123) [reply bos_mint/views.py]: fixed

File: bos_mint/forms.py (hunk 2 above)
Comment (pbattu123) [reply bos_mint/forms.py]: fixed

File: bos_mint/forms.py (hunk 3 above)
Comment (pbattu123) [reply bos_mint/forms.py]: Will refine while fixing BOS-55 in next sprint

**Pull request merged**